*: rename pkg/kvserver/storage{base,pb} to pkg/kv/kvserver/kvserver{base,pb}

irfansharif opened the pull request:
Not sure if there was a reason we didn't already do this.

``` 
    *: rename pkg/kvserver/storagebase to pkg/kv/kvserver/kvserverbase

    Simple goland rename and `make generate`. Fall out from earlier
    pkg/storage to pkg/kv{...} rename.

    Release note: None
```
```
    *: rename pkg/kvserver/storagepb to pkg/kv/kvserver/kvserverpb

    Simple goland rename and `make generate`. Fall out from earlier
    pkg/storage to pkg/kv{...} rename.

    Release note: None 
```

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/49286)
<!-- Reviewable:end -->

knz reviewed (approved):
LGTM but you need to rebase

irfansharif committed:
*: rename pkg/kvserver/storagepb to pkg/kv/kvserver/kvserverpb

Simple goland rename and `make generate`. Fall out from earlier
pkg/storage to pkg/kv{...} rename.

Release note: None

irfansharif committed:
*: rename pkg/kvserver/storagebase to pkg/kv/kvserver/kvserverbase

Simple goland rename and `make generate`. Fall out from earlier
pkg/storage to pkg/kv{...} rename.

Release note: None

irfansharif commented:
bors r+
